modules/jaeger-config-collector.adoc

```diff
@@ -40,46 +36,4 @@ The collectors are stateless and thus many instances of Jaeger Collector can be
 |log-level:
 |Logging level for the collector.
 |`trace`, `debug`, `info`, `warning`, `error`, `fatal`, `panic`
-
-|maxReplicas:
-|Specifies the maximum number of replicas to create when autoscaling the Collector.
-|Integer, for example, `100`
-
-|num-workers:
```

Comment:
Not sure why the following 3 parameters are being removed? They are no related to auto-scaling.

Also realised this table is mixing "options", that belong "spec/collector/options", as well as parameters that belong under "spec/collector" - maybe these need to be in separate tables, to avoid confusion?

Reply:
Also, `num-workers` and `queue-size` should be scoped to `collector`, e.g.
```
|collector:
  num-workers:
```